src/renault_api/kamereon/models.py

```diff
@@ -26,7 +47,15 @@ class KamereonResponseError(BaseModel):
 
     def raise_for_error_code(self) -> None:
         """Raise exception from response error."""
-        raise KamereonResponseException(self.errorCode, self.get_error_details())
+        error_details = self.get_error_details()
+        for common_error in COMMON_ERRRORS:
+            if (
+                self.errorCode == common_error["errorCode"]
+                and error_details == common_error["error_details"]
```

Comment:
I think `error_details` should be trusted on the API side.

Checking the code is enough.

If one day the details changes, you code will not work.

Reply:
Renault's servers sometimes give the same errorcode with a different error detail. But indeed not sure if this code should differentiate on this detail level. (Probably just better to only pass that detail string along when raising the exception)